ci/playbooks/kuttl/trigger-kuttl-tests.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: 'Install {{ operator_name }} operator'
```

Comment:
I don't think this step is needed, as the kuttl targets already install all the operators needed, e.g. `make keystone_kuttl` will install mariadb and keystone operators